src/main/resources/classesToInspect

```diff
@@ -2,3 +2,6 @@ com.koverse.example.spark.WordCountTransform
 com.koverse.example.spark.WordCountDataFrameTransform
 com.koverse.example.spark.JoinTransform
 com.koverse.example.spark.JavaWordCountTransform
+com.koverse.example.spark.NaiveBayesTrainedTransform
+com.koverse.example.spark.StoreVariousDataTypesTransform
```

Comment:
This line should be removed right?

Reply:
Yeah, this should be removed com.koverse.example.spark.StoreVariousDataTypesTransform. There should be a NaiveBayesPredictTransform in there as well

Reply:
I can add the README as well